examples/cfd/external_aerodynamics/domino_sensitivity/main.py

```diff
@@ -0,0 +1,491 @@
+                )
+
+    @cached_property
+    def num_vol_vars(self) -> int:
+        return sum(
+            3 if v == "vector" else 1
+            for k, v in self.cfg.variables.volume.solution.items()
+        )
+
+    @cached_property
+    def num_surf_vars(self) -> int:
+        return sum(
+            3 if v == "vector" else 1
+            for k, v in self.cfg.variables.surface.solution.items()
+        )
+
+    @cached_property
+    def bounding_box_min_max(self) -> tuple[NDArray[np.float32], NDArray[np.float32]]:
+        """Get the minimum and maximum coordinates of the bounding box from config.
+
+        Returns:
+            tuple[NDArray[np.float32], NDArray[np.float32]]: Min and max coordinates
+
+        Raises:
+            ValueError: If min or max coordinates are not specified in config
+        """
+        try:
+            return (
+                np.array(self.cfg.data.bounding_box.min, dtype=np.float32),
+                np.array(self.cfg.data.bounding_box.max, dtype=np.float32),
+            )
+        except AttributeError:
+            raise ValueError(
+                "Config must specify both `bounding_box.min` and `bounding_box.max`"
+            )
+
+    @cached_property
+    def bounding_box_surface_min_max(
+        self,
+    ) -> tuple[NDArray[np.float32], NDArray[np.float32]]:
+        """Get the minimum and maximum coordinates of the surface bounding box from config.
+
+        Returns:
+            tuple[NDArray[np.float32], NDArray[np.float32]]: Min and max coordinates
+
+        Raises:
+            ValueError: If min or max coordinates are not specified in config
+        """
+        try:
+            return (
+                np.array(self.cfg.data.bounding_box_surface.min, dtype=np.float32),
+                np.array(self.cfg.data.bounding_box_surface.max, dtype=np.float32),
+            )
+        except AttributeError:
+            raise ValueError(
+                "Config must specify both `bounding_box_surface.min` and `bounding_box_surface.max`"
+            )
+
+    @cached_property
+    def vol_factors(self) -> torch.Tensor:
+        """
+        Computes the factors that are later used to unnormalize the volume predictions.
+
+        These are saved at training time based on statistics of the training data, and re-used at each inference call.
+        """
+        return torch.from_numpy(
+            np.array(
+                [
+                    [
+                        2.1508515,
+                        1.0027921,
+                        1.0663894,
+                        1.1288369,
+                        0.05063211,
+                        0.00381244,
+                    ],
+                    [
+                        -1.9028450e00,
+                        -1.0032533e00,
+                        -1.0505041e00,
+                        -1.4412953e00,
+                        1.5563720e-18,
+                        -2.7427445e-20,
+                    ],
```

Comment:
You say in the comment that this is meant to be saved at training time and later used to unnormalize predictions.  Should this instead be loading from a file?  Hardcoding the numbers seems like it will lead to numerical errors as soon as someone uses a different dataset.

Reply:
This is copied from analogous code in the main DoMINO inference example:

https://github.com/NVIDIA/physicsnemo/blob/9cdfe789c5bd594f2a22cee735c41911167fe2c9/examples/cfd/external_aerodynamics/domino/src/inference_on_stl.py#L751

Given that this example is really a subset of that main example, I'll keep the logic as-is here until we can fix it there first in a later PR.